clients/osv/src/main/kotlin/OsvApiClient.kt

```diff
@@ -87,7 +87,7 @@ interface OsvApiClient {
 }
 
 @Serializable
-data class VulnerabilitiesForPackageRequest private constructor(
+class VulnerabilitiesForPackageRequest private constructor(
```

Comment:
Commit message nits:
- "with invalid parameter is still" -> "with invalid parameter*s* *was* still"
- "So, a normal" -> "So, *use* a normal"